<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Progress</class>
 <widget class="QDialog" name="Progress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Pb_PauseContinue">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pb_Cancel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="Tw_Display">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>331</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="Tw_VerticalHeader">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>19</y>
     <width>21</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="Tw_HorizontalHeader">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="La_Status">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
